test/conformance/testdata/conformance.txt

```diff
@@ -28,6 +28,7 @@ test/e2e/apps/rc.go: "should adopt matching pods on creation"
 test/e2e/apps/rc.go: "should release no longer matching pods"
 test/e2e/apps/replica_set.go: "should serve a basic image on each replica with a public image"
 test/e2e/apps/replica_set.go: "should adopt matching pods on creation and release no longer matching pods"
+test/e2e/apps/replica_set.go: "should surface a failure condition on a common issue like exceeded quota"
```

Comment:
This was added manually instead of via the instructions at [test/conformance/README.md](https://github.com/kubernetes/kubernetes/blob/master/test/conformance/README.md)
```
--- test/conformance/testdata/conformance.txt	2019-03-06 02:22:27.254102886 +0000
+++ test/conformance/conformance.txt	2019-03-06 02:28:07.013808162 +0000
@@ -27,8 +27,8 @@
 test/e2e/apps/rc.go: "should adopt matching pods on creation"
 test/e2e/apps/rc.go: "should release no longer matching pods"
 test/e2e/apps/replica_set.go: "should serve a basic image on each replica with a public image"
-test/e2e/apps/replica_set.go: "should adopt matching pods on creation and release no longer matching pods"
 test/e2e/apps/replica_set.go: "should surface a failure condition on a common issue like exceeded quota"
+test/e2e/apps/replica_set.go: "should adopt matching pods on creation and release no longer matching pods"
 test/e2e/apps/statefulset.go: "should perform rolling updates and roll backs of template modifications"
 test/e2e/apps/statefulset.go: "should perform canary updates and phased rolling updates of template modifications"
 test/e2e/apps/statefulset.go: "Scaling should happen in predictable order and halt if any stateful pod is unhealthy"
See instructions in test/conformance/README.md
```

Reply:
Rebuilt the PR commit as per the procedure

Reply:
Thanks for the comment. Commit rebuilt to address the concern